client/src/pages/Profile/DetailsCard/DetailsCard.tsx

```diff
@@ -0,0 +1,70 @@
+import { Paper, Typography, Stack, Avatar, Grid } from '@mui/material';
+import LocationOnIcon from '@mui/icons-material/LocationOn';
+
+export default function DetailsCard(): JSX.Element {
+  const coverImg = 'https://images.unsplash.com/photo-1625603736199-775425d2890a';
```

Comment:
Alternatively, we could store all of these variables within a single object to increase readability